fc/fc_service_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
```

Comment:
we actually have an open PR to remove this package for good https://github.com/nutanix-cloud-native/prism-go-client/pull/236